src/routes/examples/main/react.mdx

```diff
@@ -1,3 +1,110 @@
 # React
 
-Coming soon!
+Setting up Mirage with Create React App is easy, it only requires a single configuration file.
+
+Let's get started by first installing the `miragejs` package.
```

Comment:
Let's copy the formatting from the Installation guide to keep things consistent I guess:

Make sure you have Mirage installed in your project:

```sh
# Using npm
npm install --save-dev miragejs
# Using Yarn
yarn add --dev miragejs
```